apps/e2e/src/engine-client/fullSanity.ts

```diff
@@ -399,6 +407,36 @@ async function fullSanity(context: RunContext) {
     subaccountName: 'default',
   });
   prettyPrint('Subaccount info after withdraw collateral', subaccountInfoAtEnd);
+
+  const minDepositRates = await client.getMinDepositRates();
+
+  prettyPrint('Min deposit rates', minDepositRates);
+
+  subaccountInfoAtEnd.balances.forEach((balance) => {
+    if (balance.type == ProductEngineType.SPOT) {
+      const minDepositRate =
+        minDepositRates.minDepositRates[balance.productId].minDepositRate;
+      console.log('product', balance.productId);
```

Comment:
Should we just combine all of these calls? `console.log` takes varargs. You could also just print out an object (can use `toPrintableObject`) - could leave rates as fractions to make life easier too